processes/release_processes/scripts/start_publish_cromwell.sh

```diff
@@ -0,0 +1,39 @@
+# Launch a Cromwell server suited for publishing new versions of Cromwell. This uses the Local backend
+# and presumes the existence of an appropriately configured MySQL instance. See
```

Comment:
ToL for the future: I know convention says to run a MySQL, but FYI one can run in-memory HSQLDB and still call cache, as long as one leaves the server running.

MySQL is more resilient across Cromwell restarts but is more steps to deal with.